File: Assignment 3 - Linked Lists/ElectricCar.py
from Car import Car
class ElectricCar(Car):
    def __init__(self, makeAndModel = "none assigned", numberOfDoors = 4, maximumNumberOfPassengers = 5, batterySize = -1):
        super().__init__(makeAndModel, numberOfDoors, maximumNumberOfPassengers)
        self.batterySize = batterySize

    def getBatterySize(self):
        return self.batterySize
    
    def setBatterySize(self, battery):
        self.batterySize = battery

    def __str__(self) -> str:
        return f"{super().__str__()} \nBattery Size: {str(self.batterySize)} \nELECTRIC CAR"
if __name__ == "__main__":
    electricCar1 = ElectricCar()
    electricCar1.setBatterySize(10)
    print(electricCar1.getBatterySize())
    print(electricCar1)

File: Assignment 4 - Lists in Java/ElectricCar.java

public class ElectricCar extends Car 
{
    private double batterySize;
    public ElectricCar(String makeAndModel, int maximumNumberOfPassengers, int numberOfDoors, double batterySize)
    {
        super(makeAndModel, maximumNumberOfPassengers, numberOfDoors);
        this.batterySize = batterySize;
    }
    public double getBatterySize()
    {
        return this.batterySize;
    }
    public void setBatterySize(double b)
    {
        this.batterySize = b;
    }
    public String toString()
    {
        return super.toString() + "\n Battery Size: " + this.batterySize;
    }
}
